.github/workflows/generate-tag.yaml

```diff
@@ -21,7 +21,6 @@ on:
         default: 'auto'
         type: choice
         options:
-          - 'patch'
```

Comment:
Curious why this option is removed? Won't having patch version releases be super nice?

Reply:
Thats related to some CI issues we ran into arrow. So basically to publish a patch release we should use auto + final instead of patch + final. 

It may sound more intuitiv, but the later will jump a release e.g: 1.0.1 to 1.0.3

Thats why these are being removed in all arrow libs

Reply:
That makes sense! Thanks @i-walker  👍